vrk-kpa/suomidigi
# SUOPA-787 Landing page hero

To test: 
- `make fresh drush-cr`
- Open mobile proto: https://xd.adobe.com/view/1067743e-f959-428d-664c-974d183bc243-6656/screen/6ebc7d4f-589d-4531-a8b9-ed6167f8c58b/Suomidigi-etusivu-mobile-1 and  desktop proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/9c11d77d-86f9-4f29-9997-ddc12c3d0df1/Suomidigi-2020-osioetusivu-XL/
- Go to a landing page f.e. https://suomidigi.docker.sh/ryhmat-ja-verkostot and edit it
- Click Hero checkbox and add new Hero paragraph ![image](https://user-images.githubusercontent.com/1712902/72720758-528dca00-3b83-11ea-9514-9e724bf263f6.png)
- Add Background image, Title, Description and Link (you may copy them from the proto)
- Save the node and check that the layout works as in the protos.
